src/py42/exceptions.py

```diff
@@ -8,10 +8,7 @@ class Py42Error(Exception):
 class Py42ArchiveFileNotFoundError(Py42Error):
     """An exception raised when a resource file is not found or the path is invalid."""
 
-    def __init__(self, device_guid, file_path):
-        message = u"File not found in archive " u"for device {0} at path {1}".format(
-            device_guid, file_path
-        )
+    def __init__(self, message):
```

Comment:
This init does not need to be defined

Reply:
I don't understand this change. When we raise the exception, we have to repeat the same exact message each time. Why not just share the code for creating the message and keep it here?  If you want to override the message, you can subclass this exception. That way we don't have to repeat code!

I don't know what you mean about it being consistent with the other exceptions. Other exceptions craft the message inside `super.__init__()`

Reply:
Agreed. Reverted the changes as requested.

 In context to consistency I was referring to classes `Py42SecurityPlanConnectionError` and `Py42StorageSessionInitializationError`, which just take an error message and were identical to this change.